test/test_data_frame_variable_operation.rb

```diff
@@ -439,6 +439,14 @@ class DataFrameVariableOperationTest < Test::Unit::TestCase
       assert_equal str2, @df2.assign { assigner2.to_a }.tdr_str
     end
 
+    sub_test_case 'Dataframe with zero n_records' do
+      test 'assign by block preserves types' do
```

Comment:
We don't need expectation in test name. It's implemented in test. We can just use what is tested in test name.

```suggestion
      test 'assign by block' do
```